diff --git a/node_modules/@react-native/community-cli-plugin/node_modules/metro-config/src/defaults/validConfig.js b/node_modules/@react-native/community-cli-plugin/node_modules/metro-config/src/defaults/validConfig.js
index fb0c27b..7c9c907 100644
--- a/node_modules/@react-native/community-cli-plugin/node_modules/metro-config/src/defaults/validConfig.js
+++ b/node_modules/@react-native/community-cli-plugin/node_modules/metro-config/src/defaults/validConfig.js
@@ -8,6 +8,7 @@ async function validConfig() {
   const defaultConfig = await require("./index").default("/path/to/project");
   const validConfig = {
     ...defaultConfig,
+    fileMapCacheDirectory: "/path/to/cache",
     resolver: {
       ...defaultConfig.resolver,
       resolveRequest: function CustomResolver() {
