diff --git a/node_modules/@react-native-async-storage/async-storage/android/build.gradle b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
index 4f80202..462d51b 100644
--- a/node_modules/@react-native-async-storage/async-storage/android/build.gradle
+++ b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
@@ -24,6 +24,10 @@ apply plugin: "com.facebook.react"
 // ksp/room is required to support legacy storage
 apply plugin: 'com.google.devtools.ksp'
 
+kotlin {
+    jvmToolchain(21)
+}
+
 
 android {
     namespace "org.asyncstorage"
@@ -54,6 +58,10 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
     sourceSets {
         main {
             java.srcDirs += [
@@ -78,6 +86,7 @@ android {
 }
 
 repositories {
+    maven { url = uri("$projectDir/local_repo") }
     mavenCentral()
     google()
 }
